docs/api/covidcast-signals/covid-act-now.md

```diff
@@ -10,8 +10,8 @@ nav_order: 2
 
 * **Source name:** `covid-act-now`
 * **Earliest issue available:** 2021-03-25
-* **Number of data revisions since 19 May 2020:** 0
-* **Date of last change:** Never
+* **Number of data revisions since 19 May, 2020:** 0 (see [data revision docs](../covidcast_changelog.md#covid-act-now))
+* **Date of last data revision:** Never (see [data revision docs](../covidcast_changelog.md#covid-act-now))
```

Comment:
It is redundant and feels potentially confusing to have two hyperlinks to the same reference only one line apart.  I dont know how married i am to the idea though.  (This also applies to ~all the other instances of this line in the other files)
```suggestion
* **Number of data revisions since 19 May, 2020:** 0
* **Date of last data revision:** Never (see [data revision docs](../covidcast_changelog.md#covid-act-now))
```

Reply:
I think you accomplished that by having both adjacent lines labeled "data revision" now.  Having two more mentions of it on top of that is overkill, IMNSHO, but im not willing to die on that hill.  ¯\\\_(ツ)_/¯